packages/dialtone-vue2/recipes/conversation_view/emoji_row/emoji_row.vue

```diff
@@ -142,9 +145,7 @@ export default {
   }
 
   &__reaction-number {
-    font-weight: var(--dt-font-weight-bold);
-    font-size: var(--dt-font-size-100);
+    font: var(--dt-typography-body-small);
```

Comment:
Add `line-height: var(--dt-font-line-height-100);` in the line after `font`.

This should get the button to exactly 24px tall.

The Emoji icon is a smaller size than the native emoji in the Figma file, the height of the button is out of whack w/ the spacing via padding and the "body-small" style.

Reply:
nice thanks.